mcsas3_cli_runner.py

```diff
@@ -113,15 +114,15 @@ def isWindows():
         default=Path(__file__).absolute().parent
         / "example_configurations"
         / "read_config_csv.yaml",
-        help="Path to the filename with the SAXS data",
+        help="Path to the config file how to read the input data",
         # required=True,
     )
     parser.add_argument(
         "-r",
         "--resultFile",
         type=lambda p: Path(p).absolute(),
         default=Path(__file__).absolute().parent / "test.nxs",
-        help="Path to the file to create and store the McSAS3 result in",
+        help="Path to the config file how to create and store the McSAS3 result in",
```

Comment:
This is not quite right, it is the file where the initial data, the intermediate data, the metadata and the end result is stored in.